tests/by-util/test_du.rs

```diff
@@ -1264,3 +1264,35 @@ fn test_du_blocksize_zero_do_not_panic() {
             ));
     }
 }
+
+#[test]
+fn test_du_inodes_blocksize_ineffective() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+    let fpath = at.plus("test.txt");
+    std::fs::File::create(&fpath)
+        .expect("cannot create test file")
+        .set_len(123_456)
+        .expect("cannot truncate test len to size");
```

Comment:
I think you can simplify it to:
```suggestion
    at.touch("test.txt");
```

Reply:
Indeed! I had tunnel vision after writing so many tests where the exact file size matters.